pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -188,47 +197,72 @@ func (r *reconciler) Reconcile(rule *CompletedRule) error {
 	var ofPriority *uint16
 
 	value, exists := r.lastRealizeds.Load(rule.ID)
+	ruleTable := r.getOFRuleTable(rule)
 	if rule.isAntreaNetworkPolicyRule() {
 		// For CNP, only release priorityMutex after rule is installed on OVS. Otherwise,
 		// priority re-assignments for flows that have already been assigned priorities but
 		// not yet installed on OVS will be missed.
-		r.priorityMutex.Lock()
-		defer r.priorityMutex.Unlock()
+		r.priorityMutex[ruleTable].Lock()
+		defer r.priorityMutex[ruleTable].Unlock()
```

Comment:
We should optimize not to look up table multiple times in the call stack.

Reply:
Done